splunk/common-files/Dockerfile

```diff
@@ -56,7 +55,8 @@ COPY [ "splunk/common-files/updateetc.sh", "/sbin/" ]
 
 # Setup users and groups
 RUN groupadd -r -g ${GID} ${SPLUNK_GROUP} \
-    && useradd -r -m -u ${UID} -g ${GID} ${SPLUNK_USER} \
+    && useradd -r -m -u ${UID} -g ${GID} -s /sbin/nologin -d ${SPLUNK_HOME} ${SPLUNK_USER} \
```

Comment:
Not sure why change home of splunk user in here?
In my understanding, $SPLUNK_HOME is /opt/splunk, but the user splunk's home is better as /home/splunk

And also this PR broken the splunk-orca in https://git.splunk.com/projects/TOOLS/repos/infra-internal-base-images/browse/splunk-orca/x64_debian_9/Dockerfile

Reply:
It seemed confusing to me to have a "SPLUNK_HOME" which was not the same as the home directory for the user splunk. I assumed this was made different inadvertently by not explicitly specifying a home to `useradd`. I don't feel strongly about it one way or the other, and am curious to learn benefits of splitting it out.